workers/unity/Packages/com.improbable.gdk.core/Subscriptions/StandardSubscriptionManagers/WorkerIdSubscriptionManager.cs

```diff
@@ -0,0 +1,42 @@
+using Improbable.Gdk.Core;
+using Unity.Entities;
+
+namespace Improbable.Gdk.Subscriptions
+{
+    [AutoRegisterSubscriptionManager]
+    public class WorkerIdSubscriptionManager : SubscriptionManager<WorkerId>
+    {
+        private readonly World world;
+
+        public WorkerIdSubscriptionManager(World world)
+        {
+            this.world = world;
+        }
+
+        public override Subscription<WorkerId> Subscribe(EntityId entityId)
+        {
+            var subscription = new Subscription<WorkerId>(this, new EntityId(0));
```

Comment:
You can cache this subscription so we don't reallocate this every time we subscribe.